src/dparse/parser.d

```diff
@@ -6354,7 +6354,7 @@ class Parser
      *     | $(LITERAL 'override')
      *     | $(LITERAL 'pure')
      *     | $(LITERAL 'ref')
-     *     | $(LITERAL '___gshared')
+     *     | $(LITERAL '__gshared')
```

Comment:
maybe this was intended? It pops up here and there, need to investigate what this is about